frontend/src/lib/layout/DetailItem.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  import CopyToClipboardButton from '$lib/components/CopyToClipboardButton.svelte';
+  import EditableText from '$lib/components/EditableText.svelte';
+  import type { ErrorMessage } from '$lib/forms';
+  import t, { type I18nKey } from '$lib/i18n';
+
+  export let title: I18nKey;
```

Comment:
not sure how I feel about making this an i18n key, while placeholder is a string and the translation is pushed to the user of this component. I think I'd rather this be the translated text already to stay consistent with our other components.